agu_oss/clean_data.py

```diff
@@ -19,7 +19,7 @@ def open_and_clean(url):
 
     """
 
-    temp_df = pd.read_csv(url,
+    temp_df = pd.reaaaadddd_csv(url,
```

Comment:
It looks like this might be a typo here @lwasser. Would you mind double-checking?